spec/controllers/listings_controller_spec.rb

```diff
@@ -47,7 +47,9 @@
pgraded screen with he private recliner seats that's really nice. Plenty of places to sit, plenty of choice of food, and nice lawn just outside. It's a nice place to hangout\",\"time\":1519263704},{\"author_name\":\"TRAVIS BREEN\",\"author_url\":\"https://www.google.com/maps/contrib/104114442892016303973/reviews\",\"language\":\"en\",\"profile_photo_url\":\"https://lh5.googleusercontent.com/-PbCYO-I3K2g/AAAAAAAAAAI/AAAAAAAAA3c/Ls0x7FEr7tY/s128-c0x00000000-cc-rp-mo/photo.jpg\",\"rating\":3,\"relative_time_description\":\"2 weeks ago\",\"text\":\"I'm just not a Mall person, but- the Metreon isn't \\\"too much\\\". The Target store still amazes me, it's huge and has good stuff at a decent price. I go there sometimes three times a month. There is a Starbucks and it has plenty of places to sit and people watch, also it's not as crowded as you might expect. I have only been to the movies here once. I'm a big movie person, and even though the theaters have stadium seating, great sound, (Very loud) and special events it's almost phrohibitively expensive. Especially if you are coming with your family. With admission, snacks, parking, and depending on how far you came, it adds up. Oh, there's a 3D theater, but I haven't been. So as far as I'm concernedw.google.com/maps/contrib/107087795957463823927/reviews\",\"language\":\"en\",\"profile_photo_url\":\"https://lh4.googleusercontent.com/-wlmNSmfvVv0/AAAAAAAAAAI/AAAAAAAAAX8/SjIG0XrxWLw/s128-c0x00000000-cc-rp-mo-ba3/photo.jpg\",\"rating\":4,\"relative_time_description\":\"in the last week\",\"text\":\"One of the nicer modern theaters in the city.  Prices are typically pricey, but they do accept Movie Pass which is a plus!  It’s not too far from Powell BART station and ... if you were a depraved human, there is a Target right next door where you could theoretically load up on snacks... but you would be stealing away profits from the movie theater.  Yes it would be wrong to buy a gallon sized bag of M&Ms for $5 when you could buy a tiny box of M&Ms at the movie theater for $15. So wrong!\",\"time\":1520801942},{\"author_name\":\"Sir Christian R. Hadsell\",\"author_url\":\"https://www.google.com/maps/contrib/114715729089112823454/reviews\",\"language\":\"en\",\"profile_photo_url\":\"https://lh6.googleusercontent.com/-5c-_Y2piB1g/AAAAAAAAAAI/AAAAAAAAIME/34o1-QCWERg/s128-c0x00000000-cc-rp-mo/photo.jpg\",\"rating\":4,\"relative_time_description\":\"2 months ago\",\"text\":\"Movie theater is awesome. Seating is hand picked via computer so you don't have to worry about getting in early to snag good seats... I still don't know how I feel about this. That's one part of the excitement of going. Techie carp.\",\"time\":1515434038}],\"scope\":\"GOOGLE\",\"types\":[\"shopping_mall\",\"point_of_interest\",\"establishment\"],\"url\":\"https://maps.google.com/?cid=1501807273327402770\",\"utc_offset\":-420,\"vicinity\":\"135 4th Street, San Francisco\",\"website\":\"http://www.shoppingmetreon.com/\",\"html_attributions\":[]}" }
```

Comment:
I think I might put this in `spec/fixtures` as a `.json` file and read it in here using `File.read`.

Reply:
Great. I was gonna tag you and say "Sorry?"